.github/workflows/maestro_release.yaml

```
@@ -37,8 +39,16 @@ jobs:
           python-version: "3.12"
       - name: Install uv
         uses: astral-sh/setup-uv@v6
+      - name: Install dependencies
+        run: |
+          uv pip install -e . requests
```

ah, you need `requests` for the script, in that case you don't have to install all of maestro just this will do
```suggestion
          uv pip install requests
```